cdk_typescript/src/detectors/typescript-cdk-auto-scaling-group-scaling-notifications/typescript-cdk-auto-scaling-group-scaling-notifications_compliant.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=typescript-cdk-auto-scaling-group-scaling-notifications@v1.0 defects=0}
```

Comment:
pls make sure the rule version same as the rule registry `typescript-cdk-auto-scaling-group-scaling-notifications:0.1`, check for other places